packages/alpine/build.sh

```diff
@@ -41,4 +42,14 @@ termux_step_post_configure() {
 	$CC_FOR_BUILD help_h_gen.c -o help_h_gen
 	touch -d "next hour" help_c_gen help_h_gen
 }
+termux_step_create_debscripts () {
+
+        echo "#!$TERMUX_PREFIX/bin/sh" >> postinst
+	echo "if [ ! -e $TERMUX_ANDROID_HOME/.alpine-smime ] && [ ! -e $TERMUX_ANDROID_HOME/.pine-passfile ]; then" >> postinst
```

Comment:
I'm not sure it's necessary to test for existing `.pine-passfile`: the user might have copied this already. I would say it's only necessary to check they don't already have a master password.